### PR TITLE
Handle tokens that do not grant access.

### PR DESCRIPTION
**Pull request by maciekbanas**

### CODE REVIEW

**Comment by maciekbanas** [comment]
@Cotau I added handling tokens when running local tests (after Karolina's @marcinkowskak issue #252) and merged branch on adding explicite `api` and `web` urls to repositories (#246)

File: devel/example_workflow.R
Comment (Cotau) [inline devel/example_workflow.R:1]: in file example_workflow_Roche there is an issue when calling `get_repos` function, propably not updated arguments after code changes
`Error in get_repos(., by = "team") : unused argument (by = "team")`

Comment (Cotau) [reply devel/example_workflow.R:1]: other functions also seem not to be updated with recent code changes in this file

**Review by Cotau — APPROVED** [review]
Lgtm, just update example_workflow_Roche.R file

File: devel/example_workflow.R
Comment (maciekbanas) [reply devel/example_workflow.R:1]: Thanks, I'll set an issue for that.

**Pull request merged**